### ISSUE TITLE
expiration_time_millis not updated after token refresh

### ISSUE THREAD

**Issue by billsnyder** [googleapis/google-auth-library-ruby#83]
I have written a custom token store for persisting my tokens.  Everything about automatically refreshing seems to be working and the refresh token gets persisted back to my store properly.  The issue I'm having is that the `expiration_time_millis` field on the json token passed to the `store` method of my custom token store isn't updated with the new expiration time.  It still has the old value from before the refresh happens.  This in turn leads to 2 refreshes every time the code is run and defeats the ability to actually persist tokens for reuse once the initial validity period expires.

**Comment by dazuma** [comment]
Hi. Sorry for the long delay here. We're getting caught up on our issue backlog now. Is this still an issue for you?

**Comment by billsnyder** [comment]
No problem.  I haven't tested it recently but assume it is still an issue.  I can dig into it again if that would be helpful.

**Comment by blowmage** [comment]
I believe this was fixed in the signet 0.8.0 release last year. The changelog notes:

> * Ensure the "expires_at" attribute is recalculated on refresh

The `expiration_time_millis` value in JSON is calculated from `expires_at`, and that value wasn't being updated on refresh. See googleapis/signet#79.

**Comment by blowmage** [comment]
I've just verified that the current signet does indeed address this issue.

```ruby
signet.expires_at #=> 2018-12-07 14:57:27 -0700
signet.fetch_access_token!
#=> {
#     "access_token"=>"abc123",
#     "expires_in"=>3600,
#     "token_type"=>"Bearer"
#   }
signet.expires_at #=> 2018-12-10 11:33:32 -0700
```

Therefore I am closing this issue. If you continue to see this behavior please open a new issue.
